contributors/triage/README.md

```diff
@@ -0,0 +1,13 @@
+# Kubernetes Issue Triage
+
+This directory contains Kubernetes Issue Triage Guideline and related documents.
+
+Triaging issues is a great way to get started contributing, and it will help you understand how the project is organized. Here’s a few predetermined searches for convenience:  
```

Comment:
:-) OK that's good too but let's just leave as such with use of 'are'..I believe you are OK either way. Thanks!